### DISCUSSION TITLE
Nested Routes without <NuxtPage>?

### DISCUSSION THREAD

**Discussion by ENBYSS** [nuxt/nuxt#17063] [Questions] [answered]
Hey! So when I used Nuxt2, I had routes like `/tools/something` - and this would directly go to that `something.vue` page to render it.

However, now that doesn't work. If I go into a nested page, it simply goes to `/tools` again. I went over the docs and saw that I need `<NuxtPage>` in order to render this nested page - however my problem is that I want to _unload_ `/tools` and fully navigate, rather than have a nested component appear within this page.

I'm wondering if I'm missing something here... In Nuxt 2 it was the default functionality, but in Nuxt 3 I can't find anything about replicating this in any docs. I've seen that Nuxt3 renders the paths in just the way `vue-router` accepts them for this nested functionality, but I wonder what I should do for pages where I just want a normal page, rather than nested pages.

To put it another way, currently if I try to access `/tools/something`, I get this:
```vue
<Tools>
     <Something />
</Tools>
```

Instead, I just want this:
```vue
<Something />
```

Hopefully I'm missing something and this functionality wasn't just dropped.

**Comment by danielroe** [comment, marked as answer]
Try `tools/index.vue`  and `tools/something.vue`

**Reply by ENBYSS** [reply]
Oh god, yep that makes sense.

The funny part is that I was using `index.vue` before but I switched it because I thought it was causing some unrelated bug. Thanks!
